test/wg-easy/1.0.0/templates/delete-route-post-delete-hook.yaml

```diff
@@ -0,0 +1,36 @@
+{{ $values := (. | mustDeepCopy) }}
+{{ $_ := set $values "common" (dict "nameSuffix" "wg-easy") }}
+{{ $ip := .Values.wgeasy.client_address_range | replace "x" "0" }}
+apiVersion: batch/v1
```

Comment:
We should be removing this job, this is not required anymore ?

Reply:
Had it there as fail safe mostly. But yea it's redundant